python/mxnet/cython/ndarray.pyx

```diff
@@ -13,7 +13,7 @@ cdef class NDArrayBase:
     cdef int cwritable
 
     cdef _set_handle(self, handle):
-        cdef unsigned long ptr
+        cdef unsigned long long ptr
```

Comment:
This wouldn't work on 32bit device right? Are there still any of those out there?